### PR TITLE
cli: Add many options, convert to click

### PR DESCRIPTION
**Pull request by ghost**
Before removing from Cove OCDS in
https://github.com/open-contracting/cove-ocds/pull/44

Changed JSON indent to 2 in line with
https://ocp-software-handbook.readthedocs.io/en/latest/python/code.html#id2

### CODE REVIEW

**Comment by odscjames** [comment]
I did not convert to Click in this P.R. as I was unsure about adding a dependency that could be argued was not really part of the lib - maybe some people would want that as a extra requirement? But it should be easy to convert this code to click later, when that is settled.

**Comment by ghost** [comment]
[Remove CLI from cove-ocds](https://trello.com/c/LYQf25pV/88-remove-cli-from-cove-ocds)

File: libcoveocds/cli/__main__.py [hunk 1]
```diff
@@ -1,32 +1,83 @@
...
+
+
+@click.group()
+def cli():
+    pass
+
 
+@cli.command()
+@click.argument('filename')
+@click.option('-c', '--convert', is_flag=True, help='Convert data from nested (json) to flat format (spreadsheet)')
```
Comment (Bjwebb) [inline libcoveocds/cli/__main__.py:15]: Does this also do spreadsheet->json?

Comment (odscjames) [reply libcoveocds/cli/__main__.py:15]: No, it uses ocds_json_output - like old version - so it expects JSON

**Review by Bjwebb — APPROVED** [review]

File: libcoveocds/cli/__main__.py (hunk 1 above)
Comment (ghost) [reply libcoveocds/cli/__main__.py:15]: No - it crashes. Same as old version.

**Pull request merged**